### PR TITLE
add some project-maintenance files, including LICENSE

### LINKED ISSUE #35: Add a license
Add a ``LICENSE`` file to the project. [MPL 2.0](https://choosealicense.com/licenses/mpl-2.0/) seems like a good fit, but a similar license would work.

### PR DESCRIPTION
**Pull request by djmitche**
Fixes #35.  This doesn't quite get to finishing #36, but fills out CONTIRBUTING.md is a start.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# How to Contribute
+
+We welcome pull requests from everyone. We do expect everyone to adhere to the [Mozilla Community Participation Guidelines][participation].
```
Comment (jwhitlock) [inline CONTRIBUTING.md]: The declaration of `[participation]` is missing

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## How to Report
+For more information on how to report violations of the Community Participation Guidelines, please read our '[How to Report](https://www.mozilla.org/about/governance/policies/participation/reporting/)' page.
+
+<!--
+## Project Specific Etiquette
+
+In some cases, there will be additional project etiquette i.e.: (https://bugzilla.mozilla.org/page.cgi?id=etiquette.html).
+Please update for your project.
+-->
```
Comment (jwhitlock) [inline CODE_OF_CONDUCT.md:15]: I wonder if the bot is still active that requires this verbatim comment...

File: CONTRIBUTING.md (hunk 1 above)
Comment (jwhitlock) [inline CONTRIBUTING.md]: The definition of `[participation]` is missing

**Review by jwhitlock — APPROVED** [review]
Thanks @djmitche, I didn't expect to see this PR this quickly! I really appreciate the setup notes in ``CONTRIBUTING.md``.

File: CONTRIBUTING.md (hunk 1 above)
Comment (djmitche) [reply CONTRIBUTING.md]: I forgot markdown files even *supported* that.  Will fix and push.

**Pull request merged**